------------------------------------------------------------------------
r30479 | hub | 2011-11-29 00:39:09 -0500 (Tue, 29 Nov 2011) | 2 lines
Changed paths:
   M /abiword/trunk/src/af/util/xp/ut_go_file.h

Bug 13197: only import glib.h
Patch by Dominique Leuenberger.
------------------------------------------------------------------------
r30602 | hub | 2012-01-30 02:00:38 -0500 (Mon, 30 Jan 2012) | 2 lines
Changed paths:
   M /abiword/trunk/goffice-bits/goffice/app/goffice-app.h

Bug 13247 - Fix glib headers. Patch by Funda Wang
(safe to backport)
------------------------------------------------------------------------
Index: src/af/util/xp/ut_go_file.h
===================================================================
--- ./src/af/util/xp/ut_go_file.h	(revision 30478)
+++ ./src/af/util/xp/ut_go_file.h	(revision 30479)
@@ -31,7 +31,6 @@
 
 #include <glib.h>
 #include <gsf/gsf.h>
-#include <glib/gerror.h>
 #include <time.h>
 
 G_BEGIN_DECLS
Index: goffice-bits/goffice/app/goffice-app.h
===================================================================
--- ./goffice-bits/goffice/app/goffice-app.h	(revision 30601)
+++ ./goffice-bits/goffice/app/goffice-app.h	(revision 30602)
@@ -22,7 +22,7 @@
 #ifndef GOFFICE_APP_H
 #define GOFFICE_APP_H
 
-#include <glib/gmacros.h>
+#include <glib.h>
 
 G_BEGIN_DECLS
 
